beetmoverscript/data/release_beetmover_task_schema.json

```diff
@@ -46,9 +46,12 @@
                                 "items": {
                                     "type": "string"
                                 }
+                            },
```

Comment:
Note to self: in the near future, in my backlog, I have a task to add integration tests for beetmover in such a way that we have beetmover schema for all possible beetmover jobs + actual tests against them. Like we currently have in bouncerscript or shipitscript. Right now we are reusing these two possible tasks but it's not enough.